2016-06-19 02:24:58 Error: [PDOException] SQLSTATE[HY000] [1044] Access denied for user ''@'localhost' to database 'puebaonline'
Stack Trace:
#0 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Driver\PDODriverTrait.php(47): PDO->__construct('mysql:host=loca...', 'pruebaonline', '', Array)
#1 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Driver\Mysql.php(90): Cake\Database\Driver\Mysql->_connect('mysql:host=loca...', Array)
#2 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Schema\BaseSchema.php(45): Cake\Database\Driver\Mysql->connect()
#3 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Dialect\MysqlDialectTrait.php(63): Cake\Database\Schema\BaseSchema->__construct(Object(Cake\Database\Driver\Mysql))
#4 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(52): Cake\Database\Driver\Mysql->schemaDialect()
#5 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(43): Cake\Database\Schema\Collection->__construct(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Connection.php(317): Cake\Database\Schema\CachedCollection->__construct(Object(Cake\Database\Connection), true)
#7 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(920): Cake\Database\Connection->schemaCollection()
#8 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(875): Bake\Shell\Task\ModelTask->_getAllTables()
#9 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(198): Bake\Shell\Task\ModelTask->listAll()
#10 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(124): Bake\Shell\Task\ModelTask->getAssociations(Object(Cake\ORM\Table))
#11 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(109): Bake\Shell\Task\ModelTask->getTableContext(Object(Cake\ORM\Table), 'users', 'Users')
#12 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(96): Bake\Shell\Task\ModelTask->bake('Users')
#13 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\BakeShell.php(250): Bake\Shell\Task\ModelTask->main('Users')
#14 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Collection\CollectionTrait.php(51): Bake\Shell\BakeShell->Bake\Shell\{closure}('Users', 0)
#15 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\BakeShell.php(253): Cake\Collection\Collection->each(Object(Closure))
#16 [internal function]: Bake\Shell\BakeShell->all('users')
#17 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Console\Shell.php(444): call_user_func_array(Array, Array)
#18 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(217): Cake\Console\Shell->runCommand(Array, true, Array)
#19 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(181): Cake\Console\ShellDispatcher->_dispatch(Array)
#20 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(127): Cake\Console\ShellDispatcher->dispatch(Array)
#21 C:\xampp\htdocs\pruebaonline\bin\cake.php(33): Cake\Console\ShellDispatcher::run(Array)
#22 {main}


2016-06-19 02:26:20 Error: [PDOException] SQLSTATE[HY000] [1044] Access denied for user ''@'localhost' to database 'pruebaonline'
Stack Trace:
#0 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Driver\PDODriverTrait.php(47): PDO->__construct('mysql:host=loca...', 'pruebaonline', '', Array)
#1 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Driver\Mysql.php(90): Cake\Database\Driver\Mysql->_connect('mysql:host=loca...', Array)
#2 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Schema\BaseSchema.php(45): Cake\Database\Driver\Mysql->connect()
#3 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Dialect\MysqlDialectTrait.php(63): Cake\Database\Schema\BaseSchema->__construct(Object(Cake\Database\Driver\Mysql))
#4 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(52): Cake\Database\Driver\Mysql->schemaDialect()
#5 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(43): Cake\Database\Schema\Collection->__construct(Object(Cake\Database\Connection))
#6 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Database\Connection.php(317): Cake\Database\Schema\CachedCollection->__construct(Object(Cake\Database\Connection), true)
#7 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(920): Cake\Database\Connection->schemaCollection()
#8 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(875): Bake\Shell\Task\ModelTask->_getAllTables()
#9 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(198): Bake\Shell\Task\ModelTask->listAll()
#10 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(124): Bake\Shell\Task\ModelTask->getAssociations(Object(Cake\ORM\Table))
#11 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(109): Bake\Shell\Task\ModelTask->getTableContext(Object(Cake\ORM\Table), 'users', 'Users')
#12 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(96): Bake\Shell\Task\ModelTask->bake('Users')
#13 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\BakeShell.php(250): Bake\Shell\Task\ModelTask->main('Users')
#14 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Collection\CollectionTrait.php(51): Bake\Shell\BakeShell->Bake\Shell\{closure}('Users', 0)
#15 C:\xampp\htdocs\pruebaonline\vendor\cakephp\bake\src\Shell\BakeShell.php(253): Cake\Collection\Collection->each(Object(Closure))
#16 [internal function]: Bake\Shell\BakeShell->all('users')
#17 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Console\Shell.php(444): call_user_func_array(Array, Array)
#18 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(217): Cake\Console\Shell->runCommand(Array, true, Array)
#19 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(181): Cake\Console\ShellDispatcher->_dispatch(Array)
#20 C:\xampp\htdocs\pruebaonline\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(127): Cake\Console\ShellDispatcher->dispatch(Array)
#21 C:\xampp\htdocs\pruebaonline\bin\cake.php(33): Cake\Console\ShellDispatcher::run(Array)
#22 {main}


2016-11-06 22:55:26 Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'pruebaonline.resultados' doesn't exist
Stack Trace:
#0 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\cakephp\src\Database\Schema\Collection.php(96): Cake\Database\Schema\Collection->_reflect('Column', 'resultados', Array, Object(Cake\Database\Schema\Table))
#1 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\cakephp\src\Database\Schema\CachedCollection.php(64): Cake\Database\Schema\Collection->describe('resultados', Array)
#2 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\cakephp\src\ORM\Table.php(430): Cake\Database\Schema\CachedCollection->describe('resultados')
#3 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\cakephp\src\ORM\Table.php(506): Cake\ORM\Table->schema()
#4 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(206): Cake\ORM\Table->primaryKey()
#5 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(124): Bake\Shell\Task\ModelTask->getAssociations(Object(Cake\ORM\Table))
#6 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(109): Bake\Shell\Task\ModelTask->getTableContext(Object(Cake\ORM\Table), 'resultados', 'Resultados')
#7 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\bake\src\Shell\Task\ModelTask.php(96): Bake\Shell\Task\ModelTask->bake('Resultados')
#8 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\bake\src\Shell\BakeShell.php(250): Bake\Shell\Task\ModelTask->main('Resultados')
#9 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\cakephp\src\Collection\CollectionTrait.php(51): Bake\Shell\BakeShell->Bake\Shell\{closure}('Resultados', 0)
#10 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\bake\src\Shell\BakeShell.php(253): Cake\Collection\Collection->each(Object(Closure))
#11 [internal function]: Bake\Shell\BakeShell->all('resultados')
#12 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\cakephp\src\Console\Shell.php(444): call_user_func_array(Array, Array)
#13 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(217): Cake\Console\Shell->runCommand(Array, true, Array)
#14 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(181): Cake\Console\ShellDispatcher->_dispatch(Array)
#15 C:\xampp\htdocs\pruebaonline3\vendor\cakephp\cakephp\src\Console\ShellDispatcher.php(127): Cake\Console\ShellDispatcher->dispatch(Array)
#16 C:\xampp\htdocs\pruebaonline3\bin\cake.php(33): Cake\Console\ShellDispatcher::run(Array)
#17 {main}


